tests/cupy_tests/math_tests/test_arithmetic.py

```diff
@@ -289,3 +290,21 @@ def test_modf(self, xp, dtype):
         d[0] = b
         d[1] = c
         return d
+
+
+@testing.parameterize(*testing.product({
+    'xp': [numpy, cupy],
+    'shape': [(3, 2), (), (3, 0, 2)]
+}))
+@testing.gpu
+class TestBoolSubtract(unittest.TestCase):
+
+    def test_bool_subtract(self):
+        xp = self.xp
+        if xp is numpy and not testing.numpy_satisfies('>=1.13.0'):
+            raise pytest.SkipTest('NumPy<1.13.0')
```

Comment:
How about the following?
```python
@testing.with_requires('numpy>=1.13')
@testing.numpy_cupy_raises(accept_error=TypeError)
def test_bool_subtract(self, xp):
```

It's fair to skip the cupy test with numpy<1.13.

Reply:
Hm, I felt I wanted to test its behavior even with numpy<1.13.
Is it not necessary?

Reply:
It looks an implementation detail to me whether to reproduce the old behavior if old numpy is installed.

Reply:
I don't think it's an implementation detail, because the difference affects those users using old numpy.

Reply:
Then the policy to follow newest numpy could be documented (at `cupy.subtract`?) as well as it's tested.